lib/bitcoin/lock/LockMonitor.ts

```diff
@@ -277,12 +296,8 @@ export default class LockMonitor {
     latestSavedLockInfo: SavedLockModel,
     desiredLockAmountInSatoshis: number): Promise<boolean> {

```

Comment:
Would be nice to know the returned `boolean` means here without looking at the caller pattern.